reana_server/rest/workspaces.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2017, 2018 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Reana-Server Workspaces Endpoint."""
+
+from flask import Blueprint, jsonify
+
+from reana_commons.config import ADMIN_ALLOWED_WORKSPACES, SHARED_VOLUME_PATH, DEFAULT_WORKSPACE_PATH
+
+blueprint = Blueprint("workspaces", __name__)
+
+
+@blueprint.route("/workspaces", methods=["GET"])
+def workspaces():  # noqa
+    r"""Get the list of available workspaces.
+
+    ---
+    get:
+      summary: Get the list of available workspaces.
+      operationId: workspaces
+      description: >-
+        This resource reports the available workspaces in the cluster.
+      produces:
+       - application/json
+      responses:
+        200:
+          description: >-
+            Request succeeded. The response contains the list of all workspaces.
+          schema:
+            type: object
+            properties:
+              workspaces_available:
+                type: list
```

Comment:
There is a small mistake here, it should be `array` instead of `list`. You can update reana_server `openapi` docs and publish it to `reana-commons` by doing the following:
```
cd reana_server
FLASK_APP=reana_server/app.py python ./scripts/generate_openapi_spec.py --publish
diff -q -w temp_openapi.json docs/openapi.json
mv temp_openapi.json docs/openapi.json
reana-dev git-submodule --update
```